notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb

```diff
@@ -565,7 +577,10 @@
    "cell_type": "code",
    "execution_count": 24,
    "metadata": {
-    "collapsed": false
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
```

Comment:
These are just automatic metadata changes? most of the changed notebooks look untouched besides this automatic changes